xml/deployment_prep_x86_media.xml

```diff
@@ -22,133 +22,148 @@
+       flash disk, you need to copy a DVD image to the device using the <command>dd</command>
+       command. The flash disk must not be mounted, and all data on the device
+       will be erased.
       </para>
+      <screen>&prompt.root;<command>dd</command> if=<replaceable>PATH_TO_ISO_IMAGE</replaceable> of=<replaceable>USB_STORAGE_DEVICE</replaceable> bs=4M</screen>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Booting from a USB Flash Drive</term>
+     <term>DVD</term>
      <listitem>
       <para>
-       In case your machine is not equipped with an optical drive, you can
-       boot the installation image from a flash disk. To create a bootable
-       flash disk, you need to copy a DVD image to the device using the
-       <command>dd</command> command. The flash disk must not be mounted,
-       all data on the device will be erased.
-      </para>
-      <screen>&prompt.root;<command>dd</command> if=<replaceable>PATH_TO_ISO_IMAGE</replaceable> of=<replaceable>USB_STORAGE_DEVICE</replaceable> bs=4M</screen>
-      <para>
-       The ISO image also contains the data required for the basic installation.
+       DVD media are available from &suse;, or you can make your own. This method
+       is useful if you have multiple machines to provision at the same time. It
+       requires either a built-in or removable DVD drive. The process is
+       straightforward for most computer users, but requires a lot of
+       interaction for every installation process. If you did not receive
+       a DVD, get the ISO image from the &suse; home page and burn it to a
+       blank, writable DVD.
       </para>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Booting from the Network</term>
+     <term>Network Booting</term>
      <listitem>
       <para>
-       You can only boot a computer directly from the network if this is
-       supported by the computer's firmware or BIOS. This booting method
+       If the target computer's firmware supports it, you can boot the computer
+       from the network and install from a server. This booting method
        requires a boot server that provides the needed boot images over the
        network. The exact protocol depends on your hardware. Commonly you need
        several services, such as TFTP and DHCP or PXE boot. For details, read
        <xref linkend="cha-deployment-prep-pxe"/>.
       </para>
+      <para>It is possible to install from many common network protocols, such
+       as NFS, HTTP, FTP, or SMB. For more information on how to perform such an
+       installation, refer to <xref linkend="cha-remote-installation"/>. 
+      </para>
      </listitem>
     </varlistentry>
    </variablelist>
- </sect2>
-
- <sect2 xml:id="sec-x86-media-installation">
-  <title>Installation Media</title>
-  <!--
-   taroth 2019-07-31: CAVE - same section content is used in
-   deployment_prep_aarch64_media.xml
-  -->
+   <note>
+    <title>
```

Comment:
Thematically, the content of the ```<note/>``` belongs to what is mentioned in l. 36-40 (DVD 1, DVD 2, DVD 3). How about  bringing this together and perhaps merging it? I would then also move the merged/rephrased note further up (next to the itemizedlist), because it has two advantages:

* The readers are informed that they might need network access or add-ons before they even start the procedure.

* You can keep the procedure neat and clear (because step 1 already has got additional content provided by the variablelist and adding the note clutters it up even more).

Reply:
Addressed with https://github.com/SUSE/doc-sle/pull/464/commits/e5085ae226491e2bbd7bc97f86966ad03ef14891.